src/apps/vpn/mozillavpn.h

```diff
@@ -89,6 +89,7 @@ class MozillaVPN final : public QObject {
   Q_PROPERTY(bool updating READ updating NOTIFY updatingChanged)
   Q_PROPERTY(bool stagingMode READ stagingMode CONSTANT)
   Q_PROPERTY(bool debugMode READ debugMode CONSTANT)
+  Q_PROPERTY(QString countryCode READ countryCode CONSTANT)
```

Comment:
@bakulf does this seem like an appropriate place for this property?

Reply:
It would probably be better to expose the `Location` model to QML and then you can get the client location with `VPNLocation.countryCode` amongst other details.

Reply:
Nope. This is a subscription data property. The +tax should be computed in VPNSubscriptionData